beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -3551,11 +3604,12 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         self: &Arc<Self>,
         slot: Slot,
         availability: Availability<T::EthSpec>,
+        recv: Option<Receiver<DataColumnSidecarList<T::EthSpec>>>,
```

Comment:
Can you add docs for this parameter?

Reply:
Addressed in 0a9d5a0f61b4114dff93fe88c46ca4b97dde75f2